sketches/BLE_Scratch/BLE_Scratch.ino

```diff
@@ -164,44 +191,41 @@ bool init_sensors() {
 
 void setup() {
   Serial.begin(9600);
+
 #ifdef DEBUG
-  while (!Serial)
-    ;
-  Serial.println("Started");
+  while (!Serial);
```

Comment:
What happens if we remove this check ?  because the blocking check block the code if the serial is not attached. 
could we simply go ahead and ignore the serial ?